index.html

```diff
@@ -126,10 +126,22 @@
       </p>
       <section class="informative">
         <h3>
-          Example manifest
+          Example manifests
         </h3>
         <p>
-          The following shows a typical <a>manifest</a>.
+          The following shows a very simple <a>manifest</a>.
+        </p>
+        <pre class="example json" title="very simple manifest">
+{
+  "name": "Donate App",
+  "description": "This app helps you donate to worthy causes.",
+  "icons": [{
+    "src": "images/icon.png",
+    "sizes": "192x192"
+  }]
+}
+</pre>        <p>
```

Comment:
Thanks.  I've submitted a quick commit fixing this via GitHub's web-based editor.  

If that looks good, please merge.

If not, pleas push-back and I can do the full HTML5 tidy process when I get to my desk at work.